src/Microsoft.AspNetCore.Sockets.Client/WebSocketsTransport.cs

```diff
@@ -95,23 +102,33 @@ public class WebSocketsTransport : ITransport
                 {
                     using (message)
                     {
-                        await _webSocket.SendAsync(new ArraySegment<byte>(message.Payload.Buffer.ToArray()),
+                        try
+                        {
+                            await _webSocket.SendAsync(new ArraySegment<byte>(message.Payload.Buffer.ToArray()),
                             message.MessageFormat == Format.Text ? WebSocketMessageType.Text : WebSocketMessageType.Binary, true,
                             cancellationToken);
+                        } catch(OperationCanceledException ex)
```

Comment:
nit: format
also remove `ex` because you are not using it (you actually should get a warning somewhere)